metrics/examples/prometheus/kubernetes.yaml

```diff
@@ -436,6 +439,33 @@ data:
         replacement: $1
         action: replace
 
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus-rules
+data:
+  rules.yml: |
+    groups:
+    - name: kafka
+      rules:
+      - alert: UnderReplicatedPartitions
+        expr: kafka_server_replicamanager_underreplicatedpartitions > 0
+        for: 10s
+        labels:
+          severity: warning
+        annotations:
+          summary: 'Kakfa under replicated partitions'
+          description: 'There are {{ $value }} under replicated partitions on {{ $labels.kubernetes_pod_name }}'
+      - alert: NoActiveController
```

Comment:
Are you talking about the ZooKeeperClientMetrics from Kafka or metrics coming from Zookeeper? In the first case we don't have them on the Kafka grafana dashboard. In the second case we have min, max and average latency on the Zookeeper grafana dashboard. No metrics about Zookeeper request failures ... or I didn't find them.

Reply:
I meant ZK server metrics. As for the ZK client metrics ... I think that not having them in the dashboard is not an issue ... its just about having them in Prometheus, or?

Reply:
Sure we don't need them in the dashboard at all, but I would use just the average one for latency instead of having three alarms for min, max and average. Regarding request failures I cannot see them do you mean the "outstanding" ones?

Reply:
Yeah, I guess I meant the outstanding requests. It is probably fine if you do the alert on one of them.